aws/logs_monitoring/release.sh

```diff
@@ -77,15 +77,23 @@ if [ "$PROD_RELEASE" = true ] ; then
     echo "Release aws-dd-forwarder-${VERSION} to github"
     go get github.com/github/hub
     rm -f aws-dd-forwarder-*.zip
-    zip -r aws-dd-forwarder-${VERSION}.zip .
+    zip -r aws-dd-forwarder-${VERSION}.zip . \
+        --exclude=*tools* \
```

Comment:
This is will work well enough, but would it make sense to whitelist files instead?